src/main/features/claim/views/claimant-mobile.njk

```diff
@@ -12,7 +12,7 @@
         <form method="post">
           {{ csrfProtection(csrf) }}
 
-          {{ textInput('Mobile number', 'number', form) }}
+          {{ textInput('', 'number', form) }}
```

Comment:
shouldn't there be a visually hidden label here? @radoslawn-kainos

Reply:
Yup, can't leave the field with an empty label. Setting [`labelClass`](https://github.com/hmcts/cmc-common-frontend/blob/master/macros/form.njk#L1) param to `visually-hidden` should be enough. Might require modifying `textInput` macro to pass that on.

Reply:
done